src/components/CoinCharts/index.tsx

```diff
@@ -37,6 +43,10 @@ export const CoinCharts: React.FC<CoinChartsProps> = ({
           isprice={false}
         />
       </ChartsContainer>
+      <ChartSelector
+        // handleTimeChartSelection={handleTimeChartSelection}
```

Comment:
and this?

Reply:
When my API is hooked up I will, right now I do not need.